concert/storage.py

```diff
@@ -618,14 +606,14 @@ async def write_sequence(self, name: Optional[str] = "") -> None:
         if await self.device.state() != tango.DevState.STANDBY:
             raise StorageError("Error within TangoRemoteWriter")
 
-
-    async def get_log_handler(self) -> AsyncLoggingHandlerCloser:
+    async def register_logger(self, logger_name: str, log_level: int,
+                              file_name: str) -> AsyncLoggingHandlerCloser:
         """
         Provides a logging handler for the current path, capable to facilitate
         logging at a remote host.
         """
-        await self.device.open_log_handler(self._log_name)
-        return RemoteLoggingHandler(device=self.device)
+        logger_id: str = await self.device.register_logger((logger_name, str(log_level), file_name))
```

Comment:
Here, why do we do it and not the `RemoteLoggingHandler` itself?

Reply:
We can but what's the benefit ? Some information are needed to register the logger, which we are passing in this function as arguments. We need to pass them along to the RemoteLoggingHandler init if we want it to do it. What do we gain by doing that ?